### PR TITLE
fix: Fixes issue where linter wasn’t using config file on CI

### PR DESCRIPTION
**Pull request by epau**
- Specifies the config file when running the linter on CI
- Fixes infringing files

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -27,4 +27,4 @@ jobs:
       - name: Swift Lint ClientRuntime
         uses: norio-nomura/action-swiftlint@3.1.0
```
Comment (jbelkins) [inline .github/workflows/lint.yml:28]: Latest version of this action is `3.2.1`... it's still not up to date, but maybe it gets us a more recent SwiftLint?

**Review by jbelkins — CHANGES_REQUESTED** [review]
Let's try to update the lint action, see what happens.

Other than that and fixing all the lint errors, this looks good

File: .github/workflows/lint.yml (hunk 1 above)
Comment (jbelkins) [reply .github/workflows/lint.yml:28]: https://github.com/norio-nomura/action-swiftlint/tree/3.2.1

Comment (epau) [reply .github/workflows/lint.yml]: Good idea, should be updated now!

**Review by jbelkins — APPROVED** [review]

**Review by ganeshnj — APPROVED** [review]

**Pull request merged**